types/gitdata.pp

```diff
@@ -0,0 +1,10 @@
+type Hdm::Gitdata = Array[
```

Comment:
can you add an explanation at the top like:
```
# @summary type to enforce git settings for HDM
```

That will appear in the REFERENCE.md